hazelcast/src/test/java/com/hazelcast/jet/impl/JobSummaryTest.java

```diff
@@ -128,8 +128,10 @@ public void when_streamingJobLifecycle() {
     }
 
     @Test
-    public void when_lightJob() {
+    public void when_lightJob() throws InterruptedException {
         Job job = instance.getJet().newLightJob(newStreamPipeline());
+        // wait till jobs are scanned
+        Thread.sleep(200);
 
         List<JobSummary> list = getJetClientInstanceImpl(client).getJobSummaryList();
         assertEquals(1, list.size());
```

Comment:
This will be flaky when we get overloaded Jenkins (which we do sometimes). Better to put the assertion into `assertTrueEventually`.

Reply:
Done in d9e64de72ad5a2a2d17c33c976eb9876f41dbb50